BennyTordrupVisma/Week3/Ass4/dist/bundle.js

```diff
@@ -0,0 +1,485 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
If it is addded to the global .gitignore, how do I get this change to my branch? Doing a git merge master does not change the .gitignore in the branch.